core/domain/exp_services_test.py

```diff
@@ -2627,6 +2627,13 @@ def test_loading_old_exploration_does_not_break_domain_object_ctor(self):
         # converted.
         self.assertEqual(exploration.to_yaml(), self.UPGRADED_EXP_YAML)
 
+    def test_validating_classifier_model_id_after_conversion(self):
```

Comment:
Actually, I was thinking of you adding a test here, instead:

https://github.com/Andimeo/oppia/blob/3c2333bf6160ba26c5af9a2b3abd38d4e05e6388/core/domain/exp_domain_test.py#L1019

You can follow the pattern of other migrations. Please let me know if you have any questions.